SalesforceSDK/Salesforce.Sample.HybridMainPage.Universal/www/js/index.js

```diff
@@ -0,0 +1,114 @@
+var oauth = new SalesforceJS.OAuth2();
```

Comment:
There isn't a default.js in this project.  I just started with a sample that used index, which seems appropriate for a non WinJS project.